core/src/test/scala/unit/kafka/admin/DescribeConsumerGroupTest.scala

```diff
@@ -150,4 +163,228 @@ class DescribeConsumerGroupTest extends KafkaServerTestHarness {
     consumer1Mock.stop()
     consumer2Mock.stop()
   }
+
+  @Test
+  def testDescribeNonExistingGroupWithNewConsumer() {
+    // run one consumer in the group consuming from a single-partition topic
+    val executor = new ConsumerGroupExecutor(brokerList, 1, group, topic)
+
+    // note the group to be queried is a different (non-existing) group
+    val cgcArgs = Array("--bootstrap-server", brokerList, "--describe", "--group", "missing.group")
+    val opts = new ConsumerGroupCommandOptions(cgcArgs)
+    val consumerGroupCommand = new KafkaConsumerGroupService(opts)
+
+    TestUtils.waitUntilTrue(() => {
+        try {
+          val (state, assignments) = consumerGroupCommand.describeGroup()
+          state == Some("Dead") &&
+          assignments == Some(List())
```

Comment:
`contains` seems to be not supported in Scala 2.10. And builds are failing locally for me because of that. Don't we still support 2.10? Is there a way to get around it?